global/js/main.js

```diff
@@ -801,18 +801,31 @@ var groupUtils = {
         var group = this.getGroupFromId(id);
         return `<div class="osekai__group-badge osekai__group-badge-${size}" style="--colour: ${group['Colour']}">${group['ShortName']}</div>`;
     },
-    badgeHtmlFromCommaSeperatedList: function (list, size = "small") {
-        if (list == null) return "";
+    orderBadgeArray: function(array) {
+        array = array.sort((a, b) => a.Order - b.Order)

```

Comment:
.sort sorts the array in place, it returns the same array
```suggestion
        return array.sort((a, b) => a.Order - b.Order)

```
and delete the `return array;` at the bottom